.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,26 @@
+name: Pull Request
```

Comment:
"When there is a pull request, run the Lint job which in the CI environment runs `npm run lint`"